platforms/web/vite.config.ts

```diff
@@ -80,7 +89,7 @@ export default ({ mode, command }: ConfigEnv): UserConfigExport => {
         minify: true,
         inject: {
           tags: [getGoogleVerificationTag(env), fontTags, getGtmTags(env)].flat(),
-          data: app,
+          data: { ...app, favicons },
```

Comment:
Not sure, but maybe we want to add `favicons` to `app` in the OTTConfig typing?

Reply:
I think these 2 are conceptually two separate things. Also adding typing for the favicons (a lengthy string), doesn't feel right to me. I keep it as is ;)